tests/Feature/BookTest.php

```diff
@@ -268,5 +268,37 @@ public function test_unclaimBook_validation(): void
         $response = $this->putJson('/api/books/return/1', $data);
 
         $response->assertInvalid(['email'])->assertStatus(422);
+
+       
+    public function test_addBookSuccess():void
```

Comment:
You should add another test very similar to this one, but that only sends the required fields and leaves out the optional ones